igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.groovy

```diff
@@ -245,4 +235,25 @@ class TravisBuildMonitor implements PollingMonitor{
             }
         }
     }
+
+    private void setBuildCacheTTL() {
+        /*
+        This method is here to help migrate to ttl usage. It can be removed in igor after some time.
+         */
+        buildMasters.filteredMap(BuildServiceProvider.TRAVIS).keySet().each { master ->
+            log.info "Searching for cached builds without TTL."
+
+            buildCache.getJobNames(master).each { job ->
+                Long ttl = buildCache.getTTL(master, job)
+                if (ttl == -1L) {
+                    log.info "Found build without TTL: ${master}:${job}:${ttl} - Setting TTL to ${buildCacheJobTTLSeconds()}"
+                    buildCache.setTTL(master, job, buildCacheJobTTLSeconds())
+                }
+            }
+        }
+    }
+
+    private int buildCacheJobTTLSeconds() {
+        return cachedJobTTLDays * 24 * 60 * 60
```

Comment:
I think this would read better as `return TimeUnit.DAYS.toSeconds(cachedJobTTLDays)`